docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+networks:
+  db-data-net:
+    driver: bridge
+services:
+  pythonapp:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8081:8081"
```

Comment:
The `pythonapp` service is exposing port 8081, but the `Dockerfile` exposes port 8080. Ensure that the ports match between the Dockerfile and the docker-compose file to avoid connectivity issues.